_episodes/13.md

```diff
@@ -0,0 +1,449 @@
+---
+layout: episode
+title:  "13: Trino takes a sip of Pinot!"
+date: 2021-03-18
+tags: pinot aggregation apache cube analytics
+youtube_id: ""
+wistia_id: ""
+sections: 
+   - title: "Concept of the week"
+     desc: "Data cubes and Apache Pinot"
+     time: 463
+   - title: "PR of the week"
+     desc: "PR 2028 Add Pinot connector"
+     time: 1076
+   - title: "Demo"
+     desc: "Pinot batch insertion and query using Trino Pinot Connector"
+     time: 2523
+   - title: "Question of the week"
+     desc: "Why does my passthrough query not work in the Pinot connector?"
+     time: 2523
+---
+
+Trino nation, We want to hear from you! If you have a question or pull request 
+that you would like us to feature on the show please join the 
+[Trino slack](slack.html), and go to the \#trino-community-broadcast channel 
+and let us know there. Otherwise, you can message Manfred Moser or Brian Olsen 
+directly. Also, feel free to reach out to us on our Twitter channels Brian 
+[@bitsondatadev](https://twitter.com/bitsondatadev) and Manfred 
+[@simpligility](https://twitter.com/simpligility).
+
+If you want to show us some 💕, please [give us a ⭐ on Github](/star).
+
+
+<p align="center">
+<img align="center" width="50%" height="100%" src="/assets/episode/13/trinot
+.png"/><br/>
+Commander Bun Bun loves sippin' on Pinot after a hard day of data exploration!
+</p>
+
+## Pinot links
+
+ * [Apache Pinot Slack](https://communityinviter.com/apps/apache-pinot/apache-pinot)
+ * [Pinot Meetup](https://www.meetup.com/apache-pinot/events/275991991/)
+
+## Guests
+  
+ * Xiang Fu, [Apache Pinot](https://pinot.apache.org/) PMC and Commiter and 
```

Comment:
maybe reword to 

PMC and committer at Apache Pinot, and founder of stealth mode startup

Also .. maybe we have to spell out PMC (Project Management Chair)